client/src/containers/SignupForm.jsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { Box, Button, Image, Text } from "grommet";
+import { DfModal, DfSelectField, DfTextField } from "../components";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+const SignupForm = () => {
+  const [open, setOpen] = useState(false);
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setOpen(true);
+  };
+
+  const navigate = useNavigate();
+
+  const actionButtons = [
+    {
+      label: "Cancel",
+      onClick: () => {
+        setOpen(false);
+        navigate("/error");
+      },
+    },
+    {
+      primary: true,
+      label: "Accept and Register",
+      onClick: () => {
+        setOpen(false);
+        navigate("/success");
+      },
+    },
+  ];
+
+  return (
+    <Box
+      pad="medium"
+      direction="row"
+      flex="grow"
+      align="start"
+      justify="center"
+      responsive={false}
+    >
+      <Box direction="column" align="start" width="100%">
+        <Text size="xxlarge" weight="bold" color="text-strong">
+          Sign Up
+        </Text>
+        <Text size="small" color="text-strong">
+          Sign Up for GreenLake for Data Fabric beta access
+        </Text>
+
+        <form
```

Comment:
The first time the page comes up, it shows a couple of error fields (email, country):
<img width="598" alt="image" src="https://user-images.githubusercontent.com/84544729/204341128-3ac7a9b7-8229-44d6-8587-a01582709083.png">

Initially, it it should not show any errors. Also the country seems to be set, not sure why it complains.